src/utils/contacts.js

```diff
@@ -64,11 +64,8 @@ async function removeAllEmailContactsOfUser(redisPipe, userId, exceptEmail) {
   }
 }
 
-async function replaceUserName(redisPipe, userId, verifiedEmail) {
+async function setUserName(redisPipe, userId, verifiedEmail) {
   const { config: { jwt: { defaultAudience } } } = this;
-  const internalData = await getInternalData.call(this, userId);
-  const username = internalData[USERS_USERNAME_FIELD];
-  redisPipe.hdel(USERS_USERNAME_TO_ID, username);
```

Comment:
these checks are removed, is there a reason for this? What was faulty with the previous logic?

No test cases were modified or updated, changes are not clear. If there was a bug - it needs a test case, if there was no bug before - it needs an explanation on why the logic is changed, and also a test case showcasing updated behavior